run.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+if [ "${POSTGRES_ENV_POSTGRES_PASSWORD}" == "**Random**" ]; then
+        unset POSTGRES_ENV_POSTGRES_PASSWORD
+fi
+
+POSTGRES_HOST=${POSTGRES_PORT_5432_TCP_ADDR:-${POSTGRES_HOST}}
```

Comment:
these variables e.g. `POSTGRES_PORT_5432_TCP_ADDR`, is this a documented Docker feature that they show up  / are exported to the other containers, or is it rather flow.ch-specific?

I think we can get rid of these, and assume we always pass in the whole set of ENV variables.

Reply:
Removed